src/core/xccl_context.c

```diff
@@ -74,65 +79,75 @@ xccl_status_t xccl_context_create(xccl_lib_h lib,
     return XCCL_OK;
 }
 
-xccl_status_t xccl_context_progress(xccl_context_h context)
-{
+xccl_status_t xccl_context_progress(xccl_context_h context) {
     xccl_tl_context_t *tl_ctx;
-    xccl_status_t     status;
-    int               i;
+    xccl_status_t team_progress_status, xccl_progress_status;
```

Comment:
why 2 different status, and not just "status" ?

Reply:
fixed